pkg/neotest/coverage.go

```diff
@@ -174,6 +180,12 @@ func processCover() map[documentName][]*coverBlock {
 			documentSeqPoints := documentSeqPoints(di, documentName)
 
 			for _, point := range documentSeqPoints {
+				// Don't track RET statements since their bounds overlap the whole function
+				// which break the resulting coverage report.
+				_, isRET := scriptRawCoverage.methodEndOffsets[uint16(point.Opcode)]
+				if isRET {
+					continue
```

Comment:
That was my best attempt. I also tried to keep return statements in the coverage but "shorten" their bounds to a single "end" line, but it's not displayed properly and sometimes even wrong when there's no explicit "return" statement in the code. Hence, I decided to remove these sequence points from coverage.

Reply:
Although I'm still not sure if it's a good solution because: it does not exclude those RET statements that are not in the end of the method. Although such RET statements have proper bounds. So may be it's even bug in the compiler and RET statements shouldn't actually have such method-body-width bounds.

Reply:
And if it's a compiler bug:
https://github.com/nspcc-dev/neo-go/blob/86ed214e8a53859fd85482370be20eb613c61419/pkg/compiler/codegen.go#L542-L543
Then it's not clear what are the correct bounds for this "RET" statement, because there's actually no `return` statement in the contract code itself.

Reply:
Ideally, if we keep everything as is on the compiler level, then on the neotest level we need some way to distinguish some "real" return statements that are presented in the code in the end of every method from "RET" statements that belong to void and named-return functions. Currently we can't distinguish them, hence this PR removes all RET statements that belong to the end of the function.

Reply:
And I probably also need to check that no other statements except "RET" may be excluded by this code.

Reply:
> And I probably also need to check that no other statements except "RET" may be excluded by this code.

OK, it’s not required, because our compiler always emit RET in the end of functions, and there's no function that ends in another statement.